sdk/eventhub/event-processor-host/test/eph.spec.ts

```diff
@@ -640,7 +654,7 @@ describe("EPH", function(): void {
         .then(() => {
           done();
         })
-        .catch(err => {
+        .catch((err) => {
```

Comment:
Was this `prettier` or is there a difference here that matters?

Reply:
This was all prettier.